Realm/RLMAccessor.mm

```diff
@@ -576,19 +576,25 @@ static RLMAccessorCode accessorCodeForType(char objcTypeCode, RLMPropertyType rl
     }
 }
 
+static void RLMReplaceShouldIncludeInDefaultSchemaMethod(Class cls, bool shouldInclude) {
+    Class metaClass = objc_getMetaClass(class_getName(cls));
+    IMP imp = imp_implementationWithBlock(^(Class){ return shouldInclude; });
+    class_replaceMethod(metaClass, @selector(shouldIncludeInDefaultSchema), imp, "b@:");
+}
+
 // implement the class method className on accessors to return the className of the
 // base object
 void RLMReplaceClassNameMethod(Class accessorClass, NSString *className) {
     Class metaClass = objc_getMetaClass(class_getName(accessorClass));
-    IMP imp = imp_implementationWithBlock(^{ return className; });
-    class_addMethod(metaClass, @selector(className), imp, "@:");
```

Comment:
Cool cool -- although `sharedSchema`, at least with these changes, needs to be `replaceMethod`